openretina/modules/core/base_core.py

```diff
@@ -68,6 +69,7 @@ def __init__(
         downsample_input_kernel_size: tuple[int, int, int] | None = None,
         input_padding: bool | int | str | tuple[int, int, int] = False,
         hidden_padding: bool | int | str | tuple[int, int, int] = True,
+        type='sin_cos'
```

Comment:
I think here it's worth it to be verbose and all it convolution type
```suggestion
        convolution_type: str = 'sin_cos'
```